README.md

```diff
@@ -48,24 +71,82 @@ Then update to your own private key.
+
+#### Basic
+The basic EVM examples demonstrate simple smart contracts and their interactions.
+
+- [call-contract](examples/evm/call-contract/): This example demonstrates how to relay a message from a source-chain to a destination-chain.
+- [call-contract-with-token](examples/evm/call-contract-with-token/): This example allows you to send aUSDC from a source chain to a destination chain and distribute it equally among specified accounts.
+- [send-token](examples/evm/send-token/): This example sends a supported erc-20 token from source chain to the destination chain.
+- [deposit-address](examples/evm/deposit-address/): This example demonstrates how to deposit tokens from one chain to another using a deposit address.
+
+#### Advanced
+The advanced EVM examples demonstrate more complex smart contracts and their interactions.
+
+- [call-contract-with-token-express](examples/evm/call-contract-with-token-express/): This example allows you to send aUSDC from a source chain to a destination chain and distribute it equally among specified accounts using the GMP Express feature.
+- [nft-linker](examples/evm//nft-linker/): This example sends the NFT that was originally minted at source-chain to destination-chain.
+- [send-ack](examples/evm/send-ack/): This example demonstrates a two-way messaging to send a message from source-chain to destination-chain.
+- [cross-chain-token](examples/evm/cross-chain-token/): This example demonstrates how to mint tokens on a source-chain and transfer them to a destination-chain.
+
+
+### Aptos Examples
+
+The Aptos examples demonstrate how to build and interact with smart contracts on the Cosmos network. The Aptos examples include both basic and advanced examples.
+
+#### Basic
+
+The basic Aptos examples demonstrate simple smart contracts and their interactions.
+
+- [call-contract](examples/aptos/call-contract/): This example demonstrates how to relay a  bi-directional messages between Aptos and an EVM chain.
+
+### Web Examples
+
+The web examples demonstrate how to build web-based dApps that utilize cross-chain functionality. Conveniently, all of the examples have been packaged together, making it simple to set up and run them all in one place.
+
+Check the [web examples README](examples-web/README.md) get started.
+
+## Contributors
+
+We appreciate your contribution to the growth of `Axelar Examples` project. Thank you for dedicating your time and effort towards building and improving our platform. Keep up the great work and happy hacking!🍻
+
+[![Contributors](https://contrib.rocks/image?repo=axelarnetwork/axelar-examples)](https://github.com/axelarnetwork/axelar-examples/README.md)
+
+## Stargazers
+
+[![Stargazers](https://git-lister.onrender.com/api/stars/axelarnetwork/axelar-examples?limit=20)](https://github.com/axelarnetwork/axelar-examples)
+
+[Go to top](#introduction)
```

Comment:
I don't think we need this, but maybe we could do something like:

Love Axelar? Give our repo a star star :arrow_up: ., and link to the top there.

Reply:
Updated